torch/ao/quantization/fx/prepare.py

```diff
@@ -1435,6 +1458,8 @@ def prepare(
 
     assert(isinstance(qconfig_mapping, QConfigMapping))
     assert(isinstance(equalization_config, QConfigMapping))
+
+    _validate_fixed_qparams_qconfigs(model, qconfig_mapping)
```

Comment:
here is where we verify that qconfig is supported by backend_config: https://github.com/pytorch/pytorch/blob/master/torch/ao/quantization/fx/prepare.py#L1199, I think we can put the validation close to this line so that it's easier to modify the logic in the future